sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorExporterBuilder.java

```diff
@@ -479,9 +461,24 @@ private HttpPipeline createHttpPipeline() {
             .build();
     }
 
-    private TelemetryItemExporter createTelemetryItemExporter() {
-        TelemetryPipeline pipeline = new TelemetryPipeline(builtHttpPipeline);
+    private StatsbeatModule initStatsbeatModule(ConfigProperties configProperties) {
+        return new StatsbeatModule(PropertyHelper::lazyUpdateVmRpIntegration);
+    }
+
+    private void startStatsbeatModule(StatsbeatModule statsbeatModule, ConfigProperties configProperties) {
+        statsbeatModule.start(
+            builtTelemetryItemExporter,
+            this::getStatsbeatConnectionString,
+            getConnectionString(configProperties)::getInstrumentationKey,
```

Comment:
can `getConnectionString(configProperties)` return null?

Reply:
no, it will never null. it will throw an exception if it does.

Reply:
that's what the spotbug finds after i changed it from instance variable to get method.